src/Services/Air/templates/AIR_PRICE_REQ.handlebars.js

```diff
@@ -37,17 +37,31 @@ module.exports = `
                 </air:AirSegment>
                 {{/segments}}
             </air:AirItinerary>
-            {{#if platingCarrier}}
-              <air:AirPricingModifiers PlatingCarrier="{{platingCarrier}}"/>
-            {{/if}}
+            
+            <air:AirPricingModifiers
+                InventoryRequestType="DirectAccess"
```

Comment:
Should it always be DirectAccess?

Reply:
It should bes to avoid unreliable inventory. Although this does not work with 1G, this only for 1P.